akd/src/storage/memory.rs

```diff
@@ -87,6 +87,13 @@ impl Storage for AsyncInMemoryDatabase {
     async fn commit_transaction(&self) -> Result<(), StorageError> {
         // this retrieves all the trans operations, and "de-activates" the transaction flag
         let ops = self.trans.commit_transaction().await?;
+        let _epoch = match ops.last() {
+            Some(DbRecord::Azks(azks)) => Ok(azks.latest_epoch),
+            other => Err(StorageError::Other(format!(
+                "The last record in the transaction log is NOT an Azks record {:?}",
+                other
```

Comment:
What do you think about adding a specific error enum to represent what's going on here? Alternatively, should we go with `StorageError::Transaction` here?